### PR TITLE
fix API optimizer get_lr to support static graph mode

### PR DESCRIPTION
**Pull request by zhwesky2010**
### PR types

Bug fixes
### PR changes

APIs
### Describe

Fix API optimizer get_lr.

Support static graph mode.

### CODE REVIEW

File: python/paddle/optimizer/optimizer.py [hunk 1]
```diff
@@ -349,7 +349,6 @@ def set_lr(self, value):
                 },
                 stop_gradient=True)
 
-    @framework.dygraph_only
     def get_lr(self):
         """
```
Comment (littletomatodonkey) [inline python/paddle/optimizer/optimizer.py:353]: 这块是不是需要修改下注释

Comment (zhwesky2010) [reply python/paddle/optimizer/optimizer.py:353]: Done

**Review by littletomatodonkey — APPROVED** [review]

**Review by wawltor — APPROVED** [review]
LGTM

**Review by TCChenlong — APPROVED** [review]
LGTM

**Pull request merged**